runtime/extern.c

```diff
@@ -853,6 +853,9 @@ static void extern_rec(struct caml_extern_state* s, value v)
       v = Field(v, i);
       continue;
     }
+    case Cont_tag: {
+      extern_invalid_argument(s, "output_value: continuation value");
+    }
```

Comment:
note that the code style for Abstract_tag is different.
```c
    case Abstract_tag:
      extern_invalid_argument(s, "output_value: abstract value (Abstract)");
      break;
```

Reply:
I have updated the PR for consistency

Reply:
LGTM